tests/kafkatest/tests/core/round_trip_fault_test.py

```diff
@@ -92,3 +93,15 @@ def test_produce_consume_with_client_partition(self):
         workload1.wait_for_done(timeout_sec=600)
         stop1.stop()
         stop1.wait_for_done()
+
+    def test_produce_consume_with_latency(self):
+        workload1 = self.trogdor.create_task("workload1", self.round_trip_spec)
+        time.sleep(2)
```

Comment:
the sleep shouldn't be needed here, right?

Reply:
Not sure -- it copied from the above tests. Maybe this was added to reduce flakiness in the task getting creating?

Reply:
Indeed, without this sleep the test only passes about half the time.